cmake/DD4hepBuild.cmake

```diff
@@ -782,7 +782,7 @@ MACRO(DD4HEP_SETUP_GEANT4_TARGETS)
         PROPERTIES
         INTERFACE_INCLUDE_DIRECTORIES "${Geant4_INCLUDE_DIRS}"
       )
-      TARGET_LINK_LIBRARIES(CLHEP::CLHEP INTERFACE G4clhep)
+      TARGET_LINK_LIBRARIES(CLHEP::CLHEP INTERFACE Geant4::G4clhep)
```

Comment:
This only applies for Geant10.06, doesn't it?

Reply:
Good point. That's very well possible. I'll try it with a 10.5 installation and add a version check if necessary.

Reply:
It has to work down to Geant 10.2.2

Reply:
Right right. I saw. Will do.